person_capture/gui_app.py

```
@@ -11577,8 +11582,8 @@ def _try_save_wic_yuv444_color_matched_full_frame(
                 try:
                     if os.path.exists(out_path):
                         os.remove(out_path)
-                except Exception:
-                    pass
+                except OSError as exc:
+                    failures.append(f"{pix_fmt}{suffix}:cleanup_failed:{out_path}:{exc}")
```

Ruff (0.15.12)

[error] Undefined name `failures`

(F821)